bids/layout/tests/test_layout_on_examples.py

```diff
@@ -0,0 +1,43 @@
+""" Tests runs layout on bids examples and make sure all files are catched"""
```

Comment:
Ignore this file for now: I am playing around to see if we can have some tests in place for those new datatypes